cloud_governance/cloud_resource_orchestration/clouds/common/abstract_cost_over_usage.py

```diff
@@ -0,0 +1,228 @@
+import logging
```

Comment:
Is it a new abstract class that serve AWS and Azure ?

Reply:
yes